.github/workflows/lxd-image.yaml

```diff
@@ -18,7 +18,7 @@ jobs:
           SPREAD_GOOGLE_KEY: ${{ secrets.SPREAD_GOOGLE_KEY }}
         run: |
           echo $SPREAD_GOOGLE_KEY > sa.json
-          spread google-nested:tests/spread/ci
+          spread google-nested:tests/spread/ci/
```

Comment:
I'm curious, is the end slash really needed or just cosmetic?

Reply:
it's needed, otherwise spread will say no filter match